docs/build/android.md

```diff
@@ -121,6 +121,18 @@ To build on Windows with `--build_java` enabled you must also:
 * run the build from an admin window
   * the Java build needs permissions to create a symlink, which requires an admin window
 
+#### Note: Proguard rules for R8 minimization Android app builds to work
+
+There are open issues for this on the [protobuf Github project](https://github.com/protocolbuffers/protobuf/issues/6463) and [R8](https://issuetracker.google.com/issues/144631039).
```

Comment:
is ORT's issue related to protobuf as well?

Reply:
hm, yeah it might be a bit confusing to link the protobuf issue here. just wanted to refer to a similar issue originally.

will update.